examples/reference/ambientLight/ambientLight1/ambientLight1.rpde

```diff
@@ -0,0 +1,18 @@
+P3D <- "processing.opengl.PGraphics3D"
+PI <- pi
```

Comment:
As long as they are temporary and all get cleaned up later the inconsistency now is fine.